src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -52,9 +55,21 @@ public PersonCard(Person person, int displayedIndex) {
         id.setText(displayedIndex + ". ");
         name.setText(person.getName().fullName);
         phone.setText(person.getPhone().value);
-        address.setText(person.getAddress().value);
-        email.setText(person.getEmail().value);
-        schedule.setText(person.getSchedule().toString());
+        if (Address.isDefaultAddress(person.getAddress())) {
```

Comment:
Just to confirm, if the person has not set their address, the AddressBook will  say "Address not yet specified" beside the icon right? 

I was wondering if it would be better to not show the entire field including the icon instead.

Reply:
okay, done.